pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -335,33 +384,34 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 	esSecrets, err := utils.ElasticsearchSecrets(ctx, secretsToWatch, r.client)
```

Comment:
Not necessarily for this PR, but I think we can cut out all of the references to `ElasticsearchPolicyRecommendationUserSecret` - it should be going through Linseed now AFAIK, so doesn't need the ES credentials for es-gateway.